roles/setup_lvms/tasks/teardown.yml

```
@@ -0,0 +1,33 @@
+---
+- name: Delete StorageClass by name
+  kubernetes.core.k8s:
+    state: absent
+    api_version: storage.k8s.io/v1
+    kind: StorageClass
+    name: "{{ sl_storage_class }}"
+  when:
+    - sl_storage_class is defined
+    - sl_storage_class | length > 0
```

The var is always defined as is set in defaults
```suggestion
  when:
    - sl_storage_class | length > 0
```